transactionaldriver-jpa-and-tomcat/README.md

```diff
@@ -0,0 +1,27 @@
+### Description
+
+This quickstart shows how to get started with Narayana and Transactional Driver with a JPA example.
+
+### Start Tomcat
+
+You must add a $TOMCAT_HOME/bin/setenv.sh with the following content:
+`export JAVA_OPTS="-Dcom.arjuna.ats.io.narayana.jta.recovery.XAResourceRecovery1=com.arjuna.ats.internal.jdbc.recovery.BasicXARecovery\;abs://$QUICKSTART_HOME/src/main/resources/h2recoveryproperties.xml\ \;1"`
+
+Start Tomcat in the usual manner, for example:
+`$TOMCAT_HOME/bin/catalina.sh  run`
+
+### Build the app
+
+`mvn clean package`
+
+### Deploy the app
+
+`cp target/*.war apache-tomcat-7.0.78/webapps/`
```

Comment:
Should be:
``` sh
`cp target/*.war $TOMCAT_HOME/apache-tomcat-7.0.78/webapps/`
```

Also tomcat is on version 8 now so we should be referencing one of those minors; apache-tomcat-8.0 unless you are tracking the version used in openshift.